### PR TITLE
Fix h1-4 in identity ui for accessibility

### LINKED ISSUE #25674: Identity UI uses heading elements inconsistently
This was reported as an accessibility issue. See details at https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1162883

In short, we should use HTML heading elements in order, starting from `<h1>`, then `<h2>`, etc., without gaps in the sequence. There are many parts of the identity UI that don't do this. For example the "my profile" screen has `h2` then `h4`.

### PR DESCRIPTION
**Pull request by HaoK**
Fixes https://github.com/dotnet/aspnetcore/issues/25674

Basic summary is that we should use h1-h6 in sequential order, note for the manage pages, the layout uses h1-h2, so those pages start at h3.  

Will need to port these changes to scaffolding as well

### CODE REVIEW

File: src/Identity/UI/src/Areas/Identity/Pages/V4/Account/ConfirmEmailChange.cshtml [hunk 1]
```diff
@@ -4,5 +4,5 @@
     ViewData["Title"] = "Confirm email change";
 }
 
-<h2>@ViewData["Title"]</h2>
+<h1>@ViewData["Title"]</h1>
```
Comment (javiercn) [inline src/Identity/UI/src/Areas/Identity/Pages/V4/Account/ConfirmEmailChange.cshtml:7]: @HaoK this likely changes the visuals, are PMs ok with the new visual aspect?

We can use the `h2` and so on Bootstrap classes to keep the visual aspect the same while being correct about the markup.

**Review by javiercn — APPROVED** [review]
LGTM if we are ok with the visual change on the page.

**Comment by HaoK** [comment]
@blowdart can you review as well since there might be some visual diffs

**Comment by blowdart** [comment]
Can someone put together screen shots? I don't have anything set up to build new builds right now.

**Comment by HaoK** [comment]
before

After

after

**Comment by HaoK** [comment]
Pretty hard to notice if you ask me :)  I can't even really tell with the tabs side by side

**Comment by HaoK** [comment]
So the net result is we bumped up the size of the text basically across the board with these changes

**Comment by blowdart** [comment]
Manage your account grows etc. This is fine by mine to be honest. Go for it.

**Review by blowdart — APPROVED** [review]

**Pull request merged**